configs/rt_openvins_plugins.yaml

```diff
@@ -0,0 +1,9 @@
+plugin_group:
```

Comment:
Why a separate file for OV? Why not just comment out OV in rt_slam_plugins?

Reply:
I think it is better to have a separate file for OV just so people can differentiate between the two. Also it points people toward having multiple configs. Commenting things out of configs seems a bit uglier to me.

Reply:
There are 2 SLAMs x 2 integrators, so that's 4 configs. That's too much.